### PR TITLE
feat: add list of kanas used based on difficulty

### PR DESCRIPTION
**Pull request by Legyldor**
### Add list of kanas used based on difficulty

### CODE REVIEW

**Comment by Legyldor** [comment]
@gcholette can you take a look ?

**Comment by gcholette** [comment]
Thx for the pull request ❤️ 

Please fix the increased height between the two main cards (Needs to be a bit more tight for mobile (my mobile is small))

before
image

Now
image


I think the gear icon should be changed for a information bubble icon because I was already planning to add a gear icon to access settings. And maybe it could be moved next to the difficulty scale (not sure if this is a good idea tough, current placement seems fine but not totally related to the difficulty)

image

File: src/components/GridSettings.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { h } from '../resources/libs/preact.js'
+import { hiragana, katakana } from '../resources/jp/kanas.js'
+import { themedClass } from '../core/theme.js'
+import { getDifficultySubset } from '../core/logic.js'
+
+export default function GridSettings({ kind, difficulty }) {
+    let kanas = kind === 'hiragana' ?  hiragana : katakana
```
Comment (gcholette) [inline src/components/GridSettings.js]: I think this let could be a const

File: src/components/GridSettings.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { h } from '../resources/libs/preact.js'
+import { hiragana, katakana } from '../resources/jp/kanas.js'
+import { themedClass } from '../core/theme.js'
+import { getDifficultySubset } from '../core/logic.js'
+
+export default function GridSettings({ kind, difficulty }) {
```
Comment (gcholette) [inline src/components/GridSettings.js]: Please rename to something related to difficulty, like DifficultyGrid or something like that

File: src/index.js [hunk 3]
```diff
@@ -30,6 +31,7 @@ function App() {
   const [wrongAnswer, setWrongAnswer] = useState(false)
   const { question, answers } = flashcard
   const [failedAnswers, setFailedAnswers] = useState([])
+  const [displaySettings, setDisplaySettings] = useState(false)
```
Comment (gcholette) [inline src/index.js]: same thing for wording, related to difficulty

Comment (Legyldor) [reply src/index.js]: done

File: src/components/GridSettings.js (hunk 2 above)
Comment (Legyldor) [reply src/components/GridSettings.js]: done

File: src/components/GridSettings.js (hunk 1 above)
Comment (Legyldor) [reply src/components/GridSettings.js]: done

**Comment by Legyldor** [comment]
changed the target branch to dev so you can rework it with your changes

**Comment by gcholette** [comment]
Looking fine! Thx!

**Pull request merged**